### ISSUE TITLE
[Feat]: ACCESSIBLE SOCIAL ANCHORS 💡

### ISSUE THREAD

**Issue by 0xabdulkhalid** [Opentek-Org/opentek#494]
### Is your feature request related to a problem? Please describe.

- This feature is related accessibility problem

### Describe the solution you'd like.

- This solution is about adding `aria-label` for social anchors (links like Github) which are placed inside the `footer` element

- it is important to note that we want to add the `title` attribute also. This is because the `title` attribute is intended to provide additional information to screen reader users.

- The `aria-label` attribute is a way more important for providing accessibility information to screen reader users. The `aria-label` attribute is an Accessible Rich Internet Applications (ARIA) attribute that can be used to provide a label for an element. This label is then read aloud by screen readers, providing additional information about the element to the user.

- In general, the `aria-label` & `title` attributes are preferred for providing accessibility information to screen reader users.

- You can consider to assign this issue to me to make the site accessible for screen readers

### Describe alternatives you've considered.

- There's no alternative for accessibility improvements

### Screenshots (Going to make changes on these elements)

![image](https://github.com/Opentek-Org/opentek/assets/92252895/3456ce17-8e56-45e2-86cb-c2e87fd679c1)

**Comment by 0xabdulkhalid** [comment]
Hey there @sunny0625, @AnirudhPanda 👋🏻

Please make sure to assign this issue to me asap to make the work done!

Thank you.
